docs/changes/wcs/16163.bugfix.rst

```diff
@@ -0,0 +1 @@
+``DistortionLookupTable`` s (which implement ``cpdis`` and ``det2im`` projection corrections to a WCS) are now correctly aligned to the image plane, and they are handled when a WCS is sliced.
```

Comment:
The orphaned "s" looks weird and I am not 100% sure removing the whitespace would break formatting or not. Is changing to singular ok?
 
```suggestion
``DistortionLookupTable`` (which implements ``cpdis`` and ``det2im`` projection corrections to a WCS) is now correctly aligned to the image plane, and they are handled when a WCS is sliced.
```

Reply:
I think removing white space should be fine. It could also be reformulated as:
```suggestion
``DistortionLookupTable`` objects (which implement ``cpdis`` and ``det2im`` projection corrections to a WCS) are now correctly aligned to the image plane, and they are handled when a WCS is sliced.
```

Reply:
However, I think "are now correctly aligned to the image plane" is a little confusing (especially for me thinking about image _alignment_). I would like to suggest something along these lines (although it feels long and could be shortened with details):
```suggestion
Fixed a bug in ``DistortionLookupTable`` (which implement ``cpdis`` and ``det2im`` projection corrections to a WCS) due to which image coordinates were mapped incorrectly into the lookup table (~1 index offset) resulting in incorrect computation of distortions based on neighboring elements in the table.

Added support for slicing WCS objects containing ``DistortionLookupTable``  which previously were ignored.
```

I also feel last line is not a bug fix but rather an enhancement and probably should be moved to a separate change log entry.

Reply:
I've tried to keep the idea but make it more succinct, thanks!

Reply:
LGTM now. Thanks!